guides/source/active_record_migrations.md

```diff
@@ -1303,6 +1303,17 @@ contents of the current `db/schema.rb` or `db/structure.sql` file. If a
 migration can't be rolled back, `bin/rails db:reset` may not help you. To find
 out more about dumping the schema see [Schema Dumping and You][] section.
 
+If you need an alternative to `db:reset` that explicitly runs all migrations,
+consider using the `bin/rails db:migrate:reset` command. It is functionally
+equivalent to `bin/rails db:drop db:create db:schema:dump db:migrate`. You can
+follow that command with `bin/rails db:seed` if needed.
+
+NOTE: `bin/rails db:reset` rebuilds the database using the current schema
+snapshot (`db/schema.rb` or `db/structure.sql`), ensuring consistency with the
+application's defined schema. On the other hand, `bin/rails db:migrate:reset`
+replays all migrations from the beginning, which can lead to schema drift if,
+for example, migrations have been altered, reordered, or removed.
```

Comment:
```suggestion
NOTE: `bin/rails db:reset` rebuilds the database using , ensuring consistency with the
application's defined schema. On the other hand, `bin/rails db:migrate:reset`
replays all migrations from the beginning, which can lead to schema drift if,
for example, migrations have been altered, reordered, or removed.
```

This is a bit wordy IMO.

> NOTE: `bin/rails db:reset` rebuilds the database using `db/schema.rb`, while `bin/rails db:migrate:reset`
replays all migrations from the beginning, which can lead to schema drift if migrations have been altered, reordered, or removed.

Reply:
Shortened.